src/Components/TabsItem/TabsItem.scss

```diff
@@ -0,0 +1,56 @@
+// .tabs-item-container {
+//   display: flex;
+//   list-style-type: none;
+//   margin-top: 20px;
+//   width: 250px;
+//   text-align: center;
+//   background-color: lightgray;
+//   padding: 15px;
+
+//   &:hover {
+//     border-top: 4px solid rgb(54, 169, 208);
+
+//     .tabs-icon {
+//       color: rgb(22, 125, 159);
+//     }
+//   }
+  
+//   .title-icon-container {
+//     display: flex;
+//   }
+// }
+
+// .tabs-content {
+//   text-align: center;
+//   line-height: 20px;
+//   margin: 20px;
+// }
+
+
+.tabs {
```

Comment:
please refactor all scss and use scss variable.

Reply:
Done.